src/main/java/frc/robot/RobotContainer.java

```diff
@@ -244,6 +269,67 @@ private void configureButtonBindings() {
                 new MagicMotionPos(mainArm, 0, 0, 0),
                 Commands.waitSeconds(.5),
                 new MagicMotionAbsoluteZero(mainArm, shaftEncoder)));
+
+    controllertwo
+        .y()
+        .onTrue(
+            Commands.sequence(
+                new MagicMotionPos(mainArm, 210, 0, 0),
+                Commands.waitSeconds(.5),
+                new MagicMotionPos(mainArm, 0, 0, 0),
+                Commands.waitSeconds(.5),
+                new MagicMotionAbsoluteZero(mainArm, shaftEncoder)));
+  }
+
+  private void configureLimelightBindings() {
+    // Start button reseeds the steer motors to fix dead wheel
+    controller
```

Comment:
Looks good generally - put the common button code for `start` and `back` in a separate function, and call in one place.
DRY: 
don't repeat yourself